stable/ipfs/templates/statefulset.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
```

Comment:
this change is backward with at least [1.10](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#statefulset-v1-apps), I won't bother check for earlier...